CFC_DataCollector/trip_generator/input.json

```diff
@@ -0,0 +1,15 @@
+{
+	"radius" : "1",
+	"starting centroids" :
+	[
+		"2703 Hallmark Dr Belmont Ca",
+		"ATT Park"
+
+	],
+	"ending centroids" :
+	[
+		"UC Berkeley",
+		"Morgan Hill"
+	],
+	"number of trips" : 100
```

Comment:
Okay!

I'll check it out.

Josh

On Wed, Jul 1, 2015 at 5:05 PM, nephraim <notifications@github.com> wrote:

> In CFC_DataCollector/trip_generator/input.json
> <https://github.com/e-mission/e-mission-server/pull/146#discussion_r33737425>
> :
>
> > @@ -0,0 +1,15 @@
> > +{
> > +	"radius" : "1",
> > +	"starting centroids" :
> > +	[
> > +		"2703 Hallmark Dr Belmont Ca",
> > +		"ATT Park"
> > +
> > +	],
> > +	"ending centroids" :
> > +	[
> > +		"UC Berkeley",
> > +		"Morgan Hill"
> > +	],
> > +	"number of trips" : 100
>
> Maybe you could allow people to specify the probability of taking a trip
> to or from specific places?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/146/files#r33737425>.
>